### PR TITLE
Make compatible with OpenRia 5.1

### PR DESCRIPTION
**Pull request by sevanko90**
Remove "DomainServices" from namespace in generated code
Update OpenRia references to 5.1
Update EF reference to 6.4.4

### CODE REVIEW

**Comment by Daniel-Svensson** [comment]
@lindellhugo do you have the time to take a look at this? Looks fine

@sevanmarc would you like to also se if it is easy to add netstandard2.0 or net6 support to the server project?

**Comment by sevanko90** [comment]
I've added net6 to the server project. I got all kinds of weird reference errors adding netstandard 2.0, that didn't want to build at all. net6 seems to be fine though.

**Review by lindellhugo — APPROVED** [review]
@sevanmarc Good job, looks good. Sorry for the late answer but I have been on vacation.
I have approved the changes but got some issues in the pipeline. I will look into that.

File: OpenRiaServices.M2M/OpenRiaServices.M2M.Server.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net472</TargetFrameworks>
+    <TargetFrameworks>net472;net6.0</TargetFrameworks>
```
Comment (lindellhugo) [inline OpenRiaServices.M2M/OpenRiaServices.M2M.Server.csproj:4]:
```suggestion
    <TargetFrameworks>net472;net6.0</TargetFrameworks>
```

**Comment by lindellhugo** [comment]
@sevanmarc - Has merged the PR. 

Had to play around with some changes in the target frameworks while investigating the build problems but reverted them before merging. Sorry for that.

**Pull request merged**